tests/functional/community.js

```diff
@@ -0,0 +1,61 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/* global casper */
+'use strict';
+
+var config = require('../lib/config');
+var helpers = require('../lib/helpers');
+var path = '/contribute/';
+var url = config.base() + path;
+
+casper.test.begin('Community, Elements: ' + url, 10, function suite(test) {
```

Comment:
Suggest changing the tests to reference the name 'Contribute' instead of 'Community', to better reflect the URL?